compare-bug-version-and-git-branch.sh

```diff
@@ -81,8 +81,12 @@ BUG_VER=$(cut -d. -f1,2  <<< "${BUG_VERSION}")
 GERRIT_VER=$(sed 's/.*-//' <<< "${GERRIT_BRANCH}")
 
 if [ "${BUG_VER}" != "${GERRIT_VER}" ]; then
-    echo "BUG id ${BUG} was filed against version '${BUG_VER}', but the change is sent for '${GERRIT_BRANCH}'"
-    exit 1
+    if [ "${BUG_VER}" == "3.8" -a "${GERRIT_BRANCH}" == "release-3.8-fb" ]; then
```

Comment:
you could echo by default as you had earlier but just add a condition:

    # accepting patch for release 3.8 on the release-3.8-fb branch
    if [ "${BUG_VER}" != "3.8" ]; then
        if [ "${GERRIT_BRANCH}" != "release-3.8" -a "${GERRIT_BRANCH}" != "release-3.8-fb" ]; then
            exit 1
        fi
    fi